indico/modules/logs/controllers.py

```diff
@@ -94,7 +106,11 @@ def _process(self):
         if not filters and not metadata_query:
             return jsonify(current_page=1, pages=[], entries=[], total_page_count=0)
 
-        query = self.object.log_entries.order_by(self.model.logged_dt.desc())
+        if self.object:
+            query = self.object.log_entries
+        else:
+            query = self.model.query
```

Comment:
```suggestion
        query = self.object.log_entries if self.object else self.model.query
```

(no strong opinion though, I think both ways are fine)